catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/WebhookRepository.java

```diff
@@ -0,0 +1,414 @@
+  public boolean delete(String id) {
+    return dao.webhookDAO().delete(UUID.fromString(id)) > 0;
+  }
+
+  public static class WebhookEntityInterface implements EntityInterface<Webhook> {
+    private final Webhook entity;
+
+    public WebhookEntityInterface(Webhook entity) {
+      this.entity = entity;
+    }
+
+    @Override
+    public UUID getId() {
+      return entity.getId();
+    }
+
+    @Override
+    public String getDescription() {
+      return entity.getDescription();
+    }
+
+    @Override
+    public String getDisplayName() {
+      return entity.getName();
+    }
+
+    @Override
+    public EntityReference getOwner() {
+      return null;
+    }
+
+    @Override
+    public String getFullyQualifiedName() {
+      return entity.getName();
+    }
+
+    @Override
+    public List<TagLabel> getTags() {
+      return null;
+    }
+
+    @Override
+    public Double getVersion() {
+      return entity.getVersion();
+    }
+
+    @Override
+    public String getUpdatedBy() {
+      return entity.getUpdatedBy();
+    }
+
+    @Override
+    public Date getUpdatedAt() {
+      return entity.getUpdatedAt();
+    }
+
+    @Override
+    public EntityReference getEntityReference() {
+      return new EntityReference()
+          .withId(getId())
+          .withName(getFullyQualifiedName())
+          .withDescription(getDescription())
+          .withDisplayName(getDisplayName())
+          .withType(Entity.WEBHOOK);
+    }
+
+    @Override
+    public URI getHref() {
+      return entity.getHref();
+    }
+
+    @Override
+    public List<EntityReference> getFollowers() {
+      return null;
+    }
+
+    @Override
+    public Webhook getEntity() {
+      return entity;
+    }
+
+    @Override
+    public ChangeDescription getChangeDescription() {
+      return entity.getChangeDescription();
+    }
+
+    @Override
+    public void setId(UUID id) {
+      entity.setId(id);
+    }
+
+    @Override
+    public void setDescription(String description) {
+      entity.setDescription(description);
+    }
+
+    @Override
+    public void setDisplayName(String displayName) {}
+
+    @Override
+    public void setUpdateDetails(String updatedBy, Date updatedAt) {
+      entity.setUpdatedBy(updatedBy);
+      entity.setUpdatedAt(updatedAt);
+    }
+
+    @Override
+    public void setChangeDescription(Double newVersion, ChangeDescription changeDescription) {
+      entity.setVersion(newVersion);
+      entity.setChangeDescription(changeDescription);
+    }
+
+    @Override
+    public void setOwner(EntityReference owner) {}
+
+    @Override
+    public Webhook withHref(URI href) {
+      return entity.withHref(href);
+    }
+
+    @Override
+    public void setTags(List<TagLabel> tags) {}
+  }
+
+  /** One webhook call back per webhook subscription */
+  public class WebhookPublisher implements EventHandler<ChangeEventHolder>, LifecycleAware {
+    // Backoff timeout in seconds
+    private static final int BACKOFF_NORMAL = 0;
```

Comment:
It would be great to have these configurable through server config section in future